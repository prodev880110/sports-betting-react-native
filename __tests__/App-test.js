/**
 * @format
 */

import 'react-native';
import React from 'react';
import LiveBtn from '../APP/components/liveBtn';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<LiveBtn addEventToSelection={(a,b,c,d)=>{ console.log(a,b,c,d)}} game={{is_blocked:0,id:1232,info:{team1_name:'dfdf',team2_name:'dfdfd'}}} betSelections={{}} e={{price:2.00,}} eventMarket={{id:2323}} oddType="decimal" competition={{id:255,name:'EPL', alias:'EPSd'}} sport={{id:2111,name:'Football',alias:'Soccer'}}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
