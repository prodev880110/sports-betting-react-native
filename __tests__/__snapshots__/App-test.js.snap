// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "borderColor": "#d5d5da",
        "borderLeftWidth": 1,
        "borderStyle": "solid",
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "fontSize": 10,
        "justifyContent": "center",
        "position": "relative",
      },
    ]
  }
>
  <View
    accessible={true}
    focusable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
        "margin": 0,
        "opacity": 1,
        "padding": 0,
        "paddingHorizontal": 0,
        "paddingVertical": 0,
      }
    }
  >
    <View
      style={
        Object {
          "color": "#0167e8",
          "fontSize": 12,
          "fontWeight": "700",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#018da0",
              "display": "flex",
            },
            undefined,
          ]
        }
      >
        2.00
      </Text>
    </View>
  </View>
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 20,
        },
        Object {
          "color": "#dedee0",
          "display": "none",
        },
        Object {
          "fontFamily": "icomoon",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    î¬’
  </Text>
</View>
`;
